client/src/components/History/CurrentHistory/HistoryDetails.vue

```diff
@@ -17,6 +20,7 @@
 
 <script>
 import prettyBytes from "pretty-bytes";
+import store from "store";
```

Comment:
Instead of importing and using the `store` directly, you'll want to access it through `this.$store`, which should always be associated through the component tree for our vue components.